src/editor.js

```diff
@@ -1529,6 +1530,31 @@ export class OdooEditor extends EventTarget {
         } else {
             this._fixFontAwesomeSelection();
         }
+        const startContainer = sel.rangeCount && closestElement(sel.getRangeAt(0).startContainer);
+        const contenteditableFalseNode =
+            startContainer &&
+            !startContainer.isContentEditable &&
+            ancestors(startContainer).includes(this.dom) &&
+            startContainer.closest('[contenteditable=false]');
+        if (contenteditableFalseNode) {
+            console.log('correct range');
+            sel.removeAllRanges();
+            const range = new Range();
+            if (contenteditableFalseNode.previousSibling) {
+                range.setStart(
+                    contenteditableFalseNode.previousSibling,
+                    contenteditableFalseNode.previousSibling.length,
+                );
+                range.setEnd(
+                    contenteditableFalseNode.previousSibling,
+                    contenteditableFalseNode.previousSibling.length,
+                );
+            } else {
+                range.setStart(contenteditableFalseNode.parentElement, 0);
+                range.setEnd(contenteditableFalseNode.parentElement, 0);
+            }
+            sel.addRange(range);
+        }
```

Comment:
Considering the issue this commit is fixing is specifically related to FontAwesome, I'd like to talk with you on monday about how this code relate to `_fixFontAwesomeSelection`.

Reply:
The fix is more generic and theoretically solve more than the fontawesome issue (even if currently the problem appears only with fontawesome).
The `_fixFontAwesomeSelection` is more specifically for fixing when the selection end up in a fontawesome from a non collapsed selection.